tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -368,6 +442,15 @@ def __init__(
         self.num_fetch_requests_cur_rank = 0
         self.num_fetch_requests = 0
         self.shutdown_event = threading.Event()
+        self._disagg_gen_init_prepared_ids: Dict[
+            ResourceManagerType, set[int]] = {
+                ResourceManagerType.KV_CACHE_MANAGER: set(),
+                ResourceManagerType.SPEC_RESOURCE_MANAGER: set(),
+                ResourceManagerType.DRAFT_KV_CACHE_MANAGER: set(),
+            }
```

Flake8 (7.3.0)

[error] closing bracket does not match indentation of opening bracket's line

(E123)